src/flint/test/test.py

```diff
@@ -1584,6 +1584,212 @@ def test_pickling():
         obj2 = pickle.loads(s)
         assert obj == obj2
 
+def test_fmpz_mod():
+    from flint import fmpz_mod_ctx, fmpz, fmpz_mod
+    
+    # Context tests
+    F163 = fmpz_mod_ctx(163)
```

Comment:
There should be some tests with different moduli. Especially we know that flint itself distinguishes the smaller than `2**64` and the larger than `2**128` cases for different implementations and we need to test that those are working correctly.

The whole point of `fmpz_mod` is to go at least larger than `2**64` so we definitely need tests for that.